scripts/presubmit-tests.sh

```diff
@@ -112,6 +112,14 @@ function markdown_build_tests() {
   (( DISABLE_MD_LINTING && DISABLE_MD_LINK_CHECK )) && return 0
   # Get changed markdown files (ignore /vendor)
   local mdfiles="$(echo "${CHANGED_FILES}" | grep \.md$ | grep -v ^vendor/)"
+  # Filter out deleted files
+  for file in $mdfiles; do
```

Comment:
I suggest replacing lines 114-122 with something like:

```
local mdfiles=""
for file in $(echo "${CHANGED_FILES}" | grep \.md$ | grep -v ^vendor/); do
  # Filter out deleted files
  [[ -f "${file}" ]] && mdfiles="${mdfiles} ${file}"
done
```

Reply:
I like the suggestion. Updated.